x-pack/test/security_solution_cypress/cypress/e2e/asset_inventory/asset_inventory_page.cy.ts

```diff
@@ -23,6 +369,23 @@ const enableAssetInventory = () => {
   setKibanaSetting(SECURITY_SOLUTION_ENABLE_ASSET_INVENTORY_SETTING, true);
 };
 
+const waitForStatusReady = (retries = 5) => {
+  if (retries === 0) throw new Error('Status never became ready');
+
+  cy.intercept('GET', '/api/asset_inventory/status').as('getStatus');
+
+  cy.reload();
+  cy.wait('@getStatus', { timeout: 20000 }).then(({ response }) => {
+    if (response?.body?.status !== 'ready') {
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.wait(7000);
```

Comment:
This is the time between each retry, maybe we can reduce it to a lower number? i.e `3000`